reana-ui/src/client.js

```diff
@@ -17,8 +17,8 @@ import { stringifyQueryParams } from "~/util";
 export const CONFIG_URL = `${api}/api/config`;
 export const USER_INFO_URL = `${api}/api/you`;
 export const USER_SIGNUP_URL = `${api}/api/register`;
-export const USER_OAUTH_SIGNIN_URL = (next) =>
-  `${api}/api/oauth/login/cern_openid?${stringifyQueryParams({ next })}`;
+export const USER_OAUTH_SIGNIN_URL = (next, sso) =>
```

Comment:
Extremely minor comment: I think it might be more "natural" to first provide the name of the SSO provider and then the `next` redirection URL?

Reply:
Can you provide an example, please? Not sure I understand.